tasks/acquia.yml

```
@@ -0,0 +1,19 @@
+version: '3'
+
+tasks:
+  fetch-db:
+    desc: "Fetches a database from Acquia"
+    env:
+      DB_DIR: '{{default "/var/www/html/files/db" .DB_DIR}}'
```

For most of the sites there is only one database and for that **default** is fine. In some projects we probably will need to pull one secondary database.
```suggestion
      DB_DIR: '{{default "/var/www/html/files/db" .DB_DIR}}'
      ACQUIA_DB_NAME: '{{default "default" .DB_DIR}}'
```